test/dkg_hybridvss_SUITE.erl

```diff
@@ -44,6 +48,27 @@ asymmetric_test(Config) ->
     {G1, G2} = dkg_test_utils:generate(Curve),
     run(Module, N, F, T, Curve, G1, G2).
 
+%% placeholder name
+fake_symmetric(Config) ->
+    N = proplists:get_value(n, Config),
+    F = proplists:get_value(f, Config),
+    T = proplists:get_value(t, Config),
+    Module = proplists:get_value(module, Config),
+    Curve = 'SS512',
+    {G1, G2} = dkg_test_utils:generate(Curve),
+    ok = run_fake(Module, N, F, T, Curve, G1, G2),
```

Comment:
Why is this called twice in both of these tests?

Reply:
oops!  I was seeing if it doubled, timing-wise, and forgot to take it out.